sdk/python/kfp/components/pipeline_channel.py

```diff
@@ -68,6 +68,7 @@ def __init__(
         name: str,
         channel_type: Union[str, Dict],
         task_name: Optional[str] = None,
+        schema_version: Optional[str] = None,
```

Comment:
Align pipeline channels with the change to `InputSpec` to propagate schema_version through to compilation.